src/taskEvents/logCard.js

```diff
@@ -0,0 +1,170 @@
+import { createElement } from './utils.js';
+
+function createSummarySection(username, log, isAllTasks, createModalFunction) {
```

Comment:
It would be good if we can wrap these arguments as an object so that they could be passed in any order

Reply:
will do that